expression/integration_test.go

```diff
@@ -2180,6 +2180,7 @@ func (s *testIntegrationSuite) TestBuiltin(c *C) {
 	result.Check(testkit.Rows("2017-01-01 00:00:00.00"))
 	result = tk.MustQuery(`select cast(20170118.999 as datetime);`)
 	result.Check(testkit.Rows("2017-01-18 00:00:00"))
+	tk.MustExec(`select convert(a2.a, unsigned int) from (select cast('"9223372036854775808"' as json) as a) as a2;`)
```

Comment:
s/MustExec/MustQuery/

Otherwise, you will find you can pass this test in master too.

Reply:
> s/MustExec/MustQuery/
> 
> Otherwise, you will find you can pass this test in master too.

What's difference between this two MustXXX operation?